packages/config/src/projects/layer2s/neva.ts

```diff
@@ -0,0 +1,22 @@
+import { upcomingL2 } from './templates/upcoming'
+import { Layer2 } from './types'
+
+export const neva: Layer2 = upcomingL2({
+  id: 'neva',
+  display: {
+    name: 'Neva',
+    slug: 'neva',
+    description:
+      'Neva is an EVM-equivalent ZK-application network that provides scalability through Celestia DA and Polygon zkEVM, transforming GameFi and streamlining DeFi.',
+    purposes: ['Universal', 'Gaming'],
+    category: 'ZK Rollup',
```

Comment:
```suggestion
    category: 'Validium',
```
if it plans to use celestia